src/components/App/App.test.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import App from './App.component';
+
+describe('App', () => {
+  it('renders App component', () => {
+    render(<App />);
```

Comment:
Hmm 🤔 , this test is expecting nothing, this would increase your coverage but not really testing anything else than it is rendering with no errors